packages/shylu/shylu_dd/bddc/src/shylu_DofManager.h

```diff
@@ -220,16 +222,16 @@ template <class LO, class GO>
     }
     for (LO i=0; i<numNode; i++) {
       if (count[i] > 0) {
-        A->insertLocalIndices
-          (i, Teuchos::ArrayView<LO>(&localDofsIndex[nodeBegin[i]], count[i]));
+	A->insertLocalIndices
+	  (i, Teuchos::ArrayView<LO>(&localDofsIndex[nodeBegin[i]], count[i]));
       }
     }
     RCP<const Map> ColMap1to1 = Tpetra::createOneToOne<LO,GO,Node>(ColMap);
     A->fillComplete(ColMap1to1, NodeMap1to1);
   }
 
   static void extractDofMap(CrsMatrixGO & A,
-                            RCP<const Map> & dofMap)
+			    RCP<const Map> & dofMap)
```

Comment:
Why not just return `RCP<const Map>`, instead of taking the Map as an output argument?  It looks like it's only an output argument, not an in/out argument.

Reply:
Done. I have some confusion about when a reference is needed for RCP objects.

Reply:
There's nothing wrong with just passing RCP by value in this case.